index.js

```diff
@@ -274,24 +278,31 @@ CB.use(markdown({
     })
     .use(attrs),
 );
-CB.use(timer('CB: Markdown'));
+MS.use(timer('Markdown'));
 
 // Headings
-CB.use(headings());
-CB.use(timer('CB: Headings'));
+MS.use(headings());
+MS.use(timer('Headings'));
 
-CB.use(redirect({
+MS.use(redirect({
     '/support': 'https://support.d2iq.com',
 }));
-CB.use(timer('CB: Redirects'));
+MS.use(timer('Redirects'));
 
 // Permalinks
-CB.use(permalinks());
-CB.use(timer('CB: Permalinks'));
+MS.use(permalinks());
+MS.use(timer('Permalinks'));
 
 // Layouts
-CB.use(layouts({ engine: 'pug', cache: true }));
-CB.use(timer('CB: Layouts'));
+MS.use((...args) => {
+  // we want to use pugs cache for the duration of one (re-)build. we need to clear it inbetween to
+  // have changes to layouts propagate directly. unfortunately there's no convenience for that and we
+  // have to reach into pug and consolidate directly for now.
+  consolidate.clearCache();
+  pug.cache = {};
+  return layouts({ engine: 'pug', cache: true })(...args);
```

Comment:
we want to keep this `cache: true` as else every layout-step takes ~150ms (tested on `dcos/2.0`). unfortunately that would cache even for subsequent runs and thus ruins live-reload for layouts. so we manually clear the cache for new runs in `pug` and `consolidate` (which wraps pug and brings yet another cache 🤯).